lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1525,12 +1558,26 @@ struct FIRStmtParser : public FIRParser {
   ParseResult parseWire();
   ParseResult parseRegister(unsigned regIndent);
 
+  /// Remove the DontTouch annotation and return a valid symbol name if the
+  /// annotation exists. Ignore DontTouch that will apply only to a subfield.
```

Comment:
don't touch on a subfield seems a bad idea.  Do you have examples where chisel is doing this?